html/src/main/nl/tudelft/ti2806/monkeyrush/client/HtmlLauncher.java~

```diff
@@ -0,0 +1,31 @@
+package nl.tudelft.ti2806.monkeyrush.client;
+
+import com.badlogic.gdx.ApplicationListener;
+import com.badlogic.gdx.backends.gwt.GwtApplication;
+import com.badlogic.gdx.backends.gwt.GwtApplicationConfiguration;
+import nl.tudelft.ti2806.monkeyrush.Main;
+
+/**
+ * Main entrypoint for the html client.
+ */
+public class HtmlLauncher extends GwtApplication {
+
+	/**
+ 	 * No idea.. Silencing checkstyle.
+	 */
+        @Override
+        public GwtApplicationConfiguration getConfig() {
+		final int width = 480;
+		final int height = 320
+                return new GwtApplicationConfiguration(width, height);
+        }
+
+	/**
+ 	 * No idea.. Silencing checkstyle.

```

Comment:
Is daar een shortcut voor? Wil namelijk niet checkstyle in zn geheel uitzetten voor 1 dingentje.